lib/widgets/witnet/transactions/value_transfer/create_dialog_box/vtt_builder/01_recipient_step.dart

```diff
@@ -484,32 +488,58 @@ class RecipientStepState extends State<RecipientStep>
   }
 
   List<Widget> _buildWithdrawalAddressInput(ThemeData theme) {
+    _addressController.text = currentAccount.address;
+    setAddress(_addressController.text, validate: false);
     return [
-      LabeledFormEntry(
-          label: localization.withdrawalAddress,
-          formEntry: InputAddress(
-            route: widget.routeName,
-            errorText: _address.error,
-            styledTextController: _addressController,
-            focusNode: _addressFocusNode,
-            keyboardType: TextInputType.text,
-            inputFormatters: [WitAddressFormatter()],
-            onChanged: (String value) {
-              setAddress(value);
-            },
-            onFieldSubmitted: (String value) {
-              showAuthorization
-                  ? _authorizationFocusNode.requestFocus()
-                  : _amountFocusNode.requestFocus();
-            },
-            onTap: () {
-              _addressFocusNode.requestFocus();
-            },
-            onTapOutside: (event) {
-              _addressFocusNode.unfocus();
-            },
-            setAddressCallback: setAddress,
-          ))
+      Text(
+        localization.withdrawalAddress,
+        style: theme.textTheme.titleMedium,
+      ),
+      SizedBox(height: 8),
+      Text(
+          isStakeTransaction
+              ? localization.stakeWithdrawalAddressText
+              : localization.unstakeWithdrawalAddressText,
+          style: theme.textTheme.bodyMedium),
+      SizedBox(height: 16),
+      WithdrawerAddress(
+        route: widget.routeName,
+        errorText: _address.error,
+        styledTextController: _addressController,
+        focusNode: _addressFocusNode,
+        keyboardType: TextInputType.text,
+        inputFormatters: [WitAddressFormatter()],
+        setAddressCallback: setAddress,
+        onChanged: (_value) => {
+          _addressController.text = currentAccount.address,
+        },
+      ),
+      SizedBox(height: 4),
+    ];
+  }
+
+  List<Widget> _buildValidatorAddressSelect(ThemeData theme) {
+    _addressController.text = currentAccount.address;
+    List<SelectItem> validatorAddressesUsedInStakes = [
+      SelectItem('validator1', 'validator1'),
```

Comment:
add TODO comment to update the items with the correct values from the explorer when the endpoints are available